VideoLocker/src/main/java/org/edx/mobile/module/notification/ParseHandleHelper.java

```diff
@@ -66,40 +52,40 @@ public static void tryToSaveLanguageSetting( ){
         String currentPreferredLanguage = locale.getLanguage();
         String currentPreferredCountry = locale.getCountry();
         boolean dirty = false;
-        if (!currentPreferredLanguage.equals(savedPreferredLanguage) ) {
+        if (!currentPreferredLanguage.equals(savedPreferredLanguage)) {
             installation.put(languageKey, currentPreferredLanguage);
             dirty = true;
         }
-        if (!currentPreferredCountry.equals(savedPreferredCountry) ) {
+        if (!currentPreferredCountry.equals(savedPreferredCountry)) {
             installation.put(countryKey, currentPreferredCountry);
             dirty = true;
         }
-        if ( dirty ) {
+        if (dirty) {
             try {
                 PrefManager.AppInfoPrefManager pmanager = new PrefManager.AppInfoPrefManager(MainApplication.instance());
                 pmanager.setAppSettingNeedSyncWithParse(false);
-                installation.saveInBackground(new com.parse.SaveCallback(){
+                installation.saveInBackground(new com.parse.SaveCallback() {
                     @Override
                     public void done(ParseException e) {
-                        if ( e == null )
+                        if (e == null)
                             return;
                         PrefManager.AppInfoPrefManager pmanager = new PrefManager.AppInfoPrefManager(MainApplication.instance());
                         pmanager.setAppSettingNeedSyncWithParse(true);
                     }
                 });
-            }catch (Exception ex){
+            } catch (Exception ex) {
                 logger.error(ex);
             }
         }
     }
 
-    public static boolean hasNotificationHash(Context context, String notificationId){
+    public static boolean hasNotificationHash(Context context, String notificationId) {
         PrefManager.AppInfoPrefManager pmanager = new PrefManager.AppInfoPrefManager(context);
         String prevHashCode = pmanager.getPrevNotificationHashKey();
         pmanager.setPrevNotificationHashKey(notificationId);
-        if ( TextUtils.isEmpty(notificationId) && TextUtils.isEmpty(prevHashCode) )
+        if (TextUtils.isEmpty(notificationId) && TextUtils.isEmpty(prevHashCode))
             return true;
-        if ( notificationId != null && notificationId.equals(prevHashCode) )
+        if (notificationId != null && notificationId.equals(prevHashCode))
             return true;
         return false;
```

Comment:
This logic can be simplified as simply `return TextUtils.equals(notificationId, prevHashCode);`